### ISSUE TITLE
form doesn't submit on pressing enter

### ISSUE THREAD

**Issue by vvswps** [narayan954/smartbrain-frontend#24]
### Description

.

### Steps to reproduce

.

**Comment by narayan954** [comment]
yeah, true that. Thanks for raising the issue, and would you like to work on this and #23 together (please create a single pr for both these changes).
Thanks!
